index.html

```diff
@@ -74,10 +74,30 @@ <h2 class="custom-align" >Connect:</h2>
         
         </ul>
     </section>
+<!-- "Leave a Message" section -->
+    <section>
+        <h2>Leave a Message</h2>
+        <form name = "leave mesaage">
```

Comment:
Looks like a typo 'mesaage'  Because of this the query for the form does not work and messageForm is undefined.
You can see this by looked at the errors in the console (right-click->inspect).

Reply:
corrected the typo